src/main/scala/cromwell/binding/WdlNamespace.scala

```diff
@@ -216,7 +216,22 @@ case class WdlNamespace(ast: Ast, source: WdlSource, importResolver: ImportResol
   private def processCommandParameter(ast: Ast): ParameterCommandPart = {
     val wdlType = processWdlType(ast.getAttribute("type"))
     val name = ast.getAttribute("name").asInstanceOf[Terminal].getSourceString
-    new ParameterCommandPart(wdlType, name)
+    val prefix = ast.getAttribute("prefix") match {
```

Comment:
Bleh, this is one of those cases where Java bleeds in because `ast.getAttribute` can return `null`, but the return type is just `Ast`